src/tbp/monty/frameworks/environments/embodied_data.py

```diff
@@ -462,6 +463,7 @@ def __next__(self):
         # NOTE: terminal conditions are now handled in experiment.run_episode loop
         else:
             self._action = self.motor_system()
+            attempting_to_touch_object = False
```

Comment:
Not something that needs to be part of this PR but do you think this would be more intuitive to understand if it was called `attempting_to_find_object` (and then similarly having the code where we rotate to look for the object called `find_object` or `search_for_object`)? To me it currently seems a bit confusing since we are always trying to touch the object even if we didn't fall off.

Reply:
I changed this to `attempting_to_find_object`.